tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/object/ObjectInBalaTest.java

```diff
@@ -546,6 +546,11 @@ public void testObjectReferingTypeFromBalaNegative() {
                 "no implementation found for the method 'toString' of class 'FrameImpl'", 53, 1);
     }
 
+    @Test(description = "Test the result of the remote function in client object with inclusion")
+    public void testRemoteFunctionReturnForObjectInclusion() {
+        BRunUtil.invoke(result, "testRemoteFunctionReturnForObjectInclusion");
```

Comment:
IMO, still doesn't indicate what exactly we are trying to test? We are just trying to test an included method?

Reply:
Isn't this same as https://github.com/ballerina-platform/ballerina-lang/pull/40386/files#diff-971027c88ba42c746eff5667d7c9fe8d8f4b4acdd0b3c2da66b9fd12b78026a9R519-R521?

Reply:
Aren't we trying to test that it's valid if parameter names are the same? While we need the returned value, we aren't really testing the value? If for some reason we are updating the tests some day, it would be better to indicate that the parameters are required for the test.

I would use something like `testObjectInclusionWithMethodWithParameters`.